docs/events/api/timeline-api-update.asciidoc

```diff
@@ -0,0 +1,108 @@
+[[timeline-api-update]]
+=== Add a note to existing timeline
+
+Add a note to existing timeline or an event of existing timeline
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/note`
+
+==== Request body
+
+A JSON object defining the note content to be added.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+|`note` | <<eventNotes-obj, eventNote>> a|A note added to specific event in the timeline.
+
+* `timelineId`: the `savedObjectId` of existing timeline the note is linked to.
+* `eventId`: required if the note is linked to a specific event of a timeline.
+* `note`: note content.
+
+|Yes
+
+
+|==============================================
+
+===== Example requests
+
+*Example 1*
+
+Add a note to an existing event of a timeline:
+
+[source,console]
+--------------------------------------------------
+PATCH api/note
+{
+  "note":{
+    "eventId":"dDaPFXkB-qWtr5vqlN9o",
+    "note":"My note",
+    "timelineId":"b2c103b0-a79d-11eb-9dce-0f3114099868"
+  }
+}
+--------------------------------------------------
+
+
+*Example 2*
+
+Add a note to an existing timeline:
+
+[source,console]
+--------------------------------------------------
+PATCH api/note
+{
+  "note":{
+    "note":"My note",
+    "timelineId":"b2c103b0-a79d-11eb-9dce-0f3114099868"
+  }
+}
+--------------------------------------------------
+
+=== Pin an event to existing timeline
+
+==== Request URL
+
+`PATCH <kibana host>:<port>/api/pinned_event`
+
+==== Request body
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+|`eventId` | String |Each document has an `_id` to uniquly identifies it. |Yes
+|`pinnedEventId` | String |pinned event `savedObjectId`. |No
+|`timelineId` | String | the `savedObjectId` of existing timeline the pinned event is linked to. |Yes
```

Comment:
A few recommendations here: 
- Capitalize the "t" in "the" at the start of the sentence
- Add "an" before "existing"
- Capitalize the "t" in "Timeline"
- Remove the period at the end